backend/battle/forms.py

```diff
@@ -46,6 +46,9 @@ def clean(self):
         if 'creator' not in cleaned_data:
             raise forms.ValidationError("ERROR: You need to be logged.")
 
+        if 'opponent' not in cleaned_data:
```

Comment:
I'm not sure, but I think this can be replaced by a `required=True` on the attribute definition (`opponent = forms.EmailField()`)

Reply:
I tried this the `required=True` and it didn't any effect. So I kept this validation.